src/scoring/hpf_scoring.py

```diff
@@ -130,12 +158,12 @@ def predict(self, input_stack):
             companion_recommendation = self.folding_in(
                 input_id_set)
         else:
-            current_app.logger.error(
+            _logger.error(
                 "{} length of missing packages beyond unknow threshold value of {}".format(
                     len(missing_packages), UNKNOWN_PACKAGES_THRESHOLD))
         return companion_recommendation, package_topic_dict, list(missing_packages)
 
-    def match_manifest(self, input_id_set):
+    def match_manifest(self, input_id_set):  # pragma: no cover
```

Comment:
tbh this is the method that should be covered by unit tests

Reply:
I will undo the pragma, once the test is fixed for it. It is not working as expected for now.

Reply:
@tisnik I found that using OrdererdDict will solve the problem. I am working on that fix.